Intersect (Core)/Threading/Threaded`1.cs

```diff
@@ -1,33 +1,32 @@
-﻿using System;
-
-namespace Intersect.Threading
+﻿namespace Intersect.Threading
 {
-
-    /// <inheritdoc />
+    /// <summary>
+    /// Represents a generic base class for creating and managing new threads.
+    /// </summary>
+    /// <typeparam name="TArgument">Type of argument that will be passed to a new thread.</typeparam>
     public abstract partial class Threaded<TArgument> : Threaded
     {
-
-        /// <inheritdoc />
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Threaded{TArgument}"/> class.
+        /// </summary>
+        /// <param name="name">Name of new thread.</param>
         protected Threaded(string name = null) : base(name)
         {
         }
 
-        /// <inheritdoc />
+        /// <summary>
+        /// Overrides the <see cref="Threaded.ThreadStart"/> method.
+        /// </summary>
```

Comment:
This comment change provides zero information to the user and removes all of the original information inherited. Revert to inheritdoc.

Reply:
If you want to document it, the documentation should be in the base class.